[Feature] Fractus Nodes

Hiroshi97 opened the pull request.

Hiroshi97 commented:
@2ndtlmining Hi, just to clarify: **Fractus** is also Cumulus nodes with Thunder is false, is that correct? If so, some of data must be pending to calculate and get the Fractus from total Cumulus nodes. 

Regarding to the header, the API returns only the total of cumulus, stratus, and nimbus so we don't have any data about Fractus yet in that time to display.

2ndtlmining commented:
Thanks ill have a test and see how it goes. I dont own one of these so ill have to ask around for some help to test.

Your question: Fractus is also Cumulus nodes with Thunder is False= Yes and No :).  Fractus is a Cumulus node thus will sit within the same queue for payment but it has the Thunder flag set to True. Its basically a storage node in the Cumulus bucket. There is a 10TB requirement a bit more upload speed. So its just a different version of Cumulus if that makes sense.

The project to introduce these types of nodes was called thunder and as such made its way into the API and code.

Cool, if the API returns only Cumulus then we leave that part as is till i can see if there is an API for it. Ill ask the question so long. Let you know how testing goes

2ndtlmining commented:
Hey mate

Managed to find a wallet:

![image](https://user-images.githubusercontent.com/108513481/213943760-6a873a79-cee8-4b32-8af0-c7b1954f80fa.png)


Couple things:

1. DWS Validation: For cractus needs to be 80mbs 
2. Icon in the Payout timer think still going to Cumulus
3. Was looking at the code changes and spotted the colateral being 0, that is 1000 but not think its actually used. 

I am still doing some other testing

2ndtlmining commented:
Another small thing, in white mode think the Fractus toggle is not visible. Suspect its just white, if that cant be fixed not a big deal:

![image](https://user-images.githubusercontent.com/108513481/213955413-24d2a755-652b-461e-8f08-7b404705f357.png)

Hiroshi97 commented:
@2ndtlmining Thanks for the feedback:
1. Fixed
2. Fixed
3. I don't know what it's actually for so just set it = 0 just in case we have any other value for the FRACTUS collateral. I've set it to 1000.
4. Toggle button for Light theme: Fixed

victorquanlam commented on client/src/main/apidata.js:
```diff
@@ -186,9 +200,8 @@ export async function fetch_global_stats(walletAddress = null) {
 
   if (resTotalDonations.status == 'fulfilled') {
     const res = resTotalDonations.value;
-    store.total_donations = res;
```
@Hiroshi97 oops. You left extra quotation marks here buddy 😄

victorquanlam commented on client/src/main/apidata.js:
```diff
@@ -186,9 +200,8 @@ export async function fetch_global_stats(walletAddress = null) {
   if (resTotalDonations.status == 'fulfilled') {
     const res = resTotalDonations.value;
-    store.total_donations = res;
+    store.total_donations = `res`;
```
@Hiroshi97 oops. You left extra quotation marks here buddy 😄
